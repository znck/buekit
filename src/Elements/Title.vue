<script>
import createMixin from './title/mixin'

export default {
  name: 'Title',
  ...createMixin('title')
}
</script>

<style lang="scss" module>
@import '~bulma/sass/utilities/_all.sass';
@import '../_base.scss';
@import '~bulma/sass/elements/title.sass';
</style>
