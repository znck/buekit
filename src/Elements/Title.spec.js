import {shallow} from 'vue-test-utils'
import {createRenderer} from 'vue-server-renderer'

import Title from './Title.vue'

describe('Title.vue', () => {
	it('matches snapshot', () => {
		const wrapper = shallow(Title, {})
		const renderer = createRenderer()

		renderer.renderToString(wrapper.vm, (err, str) => {
			if (err) {
				throw new Error(err)
			}
			expect(str).toMatchSnapshot()
		})
	})
})
