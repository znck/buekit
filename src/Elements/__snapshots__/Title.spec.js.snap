// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title.vue matches snapshot 1`] = `<h1 class="title"></h1>`;
